Twig/Extension/ReadableEnumValueTwigExtension.php

```diff
@@ -43,9 +43,9 @@ public function getFilters(): array
      * @throws ValueIsFoundInFewRegisteredEnumTypesException
      * @throws ValueIsNotFoundInAnyRegisteredEnumTypeException
      *
-     * @return string|null
+     * @return int|string|null
      */
-    public function getReadableEnumValue(?string $enumValue, ?string $enumType = null): ?string
+    public function getReadableEnumValue(?string $enumValue, ?string $enumType = null)
```

Comment:
PHPStan alerted here to the fact that the return value comes from `getReadableValue`, which can return an integer as well since that's supported, right?

Reply:
or is it always a string here in Twig? Need your insight on this @fre5h

Reply:
Please add a new test variant into this data provider https://github.com/fre5h/DoctrineEnumBundle/blob/main/Tests/Twig/Extension/ReadableEnumValueTwigExtensionTest.php#L71 to check the int values are returned as excpected

Reply:
So `int` also can be returned as in `getReadableValue`

Reply:
thanks, will do 👍

Reply:
added here: https://github.com/fre5h/DoctrineEnumBundle/pull/196/commits/15b669cf761a7f44782846bd21d088053c05a40e.

I'm a bit confused how come there weren't any type errors before though 🤔 . Is nobody using `int` enums?

Reply:
I don't really know. As for me I don't use numeric enums. This feature was added by another developer, so at least somebody uses it :) But anyway, thank you for your updates